src/test/java/org/jabref/model/entry/BibEntryWithKeywordsTest.java

```diff
@@ -0,0 +1,289 @@
+package org.jabref.model.entry;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.Optional;
+
+import org.jabref.model.FieldChange;
+import org.jabref.model.database.BibDatabase;
+import org.jabref.model.entry.field.SpecialField;
+import org.jabref.model.entry.field.StandardField;
+import org.jabref.model.entry.types.StandardEntryType;
+
+import com.google.common.collect.Sets;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class BibEntryWithKeywordsTest {
+
+    private BibEntry keywordEntry;
+
+    @BeforeEach
+    public void setUp() {
```

Comment:
The above test file also contains keyword tests (e.g testGetSingleKeywords). So what is the distinction?

Reply:
You are right - merged all into ine in 7c7488